.github/workflows/test.yml

```diff
@@ -4,18 +4,18 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.14.x, 1.15.x]
+        go-version: [1.14.x, 1.15.x, 1.16.x, 1.17.x]
         platform: [ubuntu-latest, windows-latest]
     runs-on: ${{ matrix.platform }}
     steps:
-    - name: Install Go
-      uses: actions/setup-go@v1
-      with:
-        go-version: ${{ matrix.go-version }}
-    - name: Checkout code
-      uses: actions/checkout@v2
-    - name: Test
-      run: go test -v ./...
-    - name: Test Blake3
-      run: go test -v ./...
-      working-directory: blake3
+      - name: Install Go
+        uses: actions/setup-go@v1
```

Comment:
While at it, can you change this to use `v2`? https://github.com/actions/setup-go#v2

Also would be good to have a `run: go version` somewhere below. I've had issues in other repositories where some environment variables weren't set correctly, and all steps were running on the same go version.

Reply:
Upgraded it, but it looks like `go version` is run in the preamble. I'll remove that step.